third_party/terraform/resources/resource_sql_database_instance.go.erb

```diff
@@ -698,8 +698,15 @@ func resourceSqlDatabaseInstanceRead(d *schema.ResourceData, meta interface{}) e
 		return err
 	}
 
-	instance, err := config.clientSqlAdmin.Instances.Get(project,
-		d.Id()).Do()
+	var instance *sqladmin.DatabaseInstance
+	err = retryTimeDuration(
+		func() error {
+				instance, err = config.clientSqlAdmin.Instances.Get(project, d.Id()).Do()
+			return err
+		},
+
+		DefaultRequestTimeout,
```

Comment:
Instead of passing in this as a default we could just move it to [the default `retry`](https://github.com/terraform-providers/terraform-provider-google/blob/master/google/utils.go#L329) and use that instead.